### ISSUE TITLE
no more data in UDP packet

### ISSUE THREAD

**Issue by mdavidsaver** [epics-base/pvAccessCPP#137]
Beginning with pvAccessCPP 6.1.0 (Base 7.0.2) users at SLAC report seeing servers and clients printing message like:

```
an exception caught while in UDP receiveThread at ../../src/remote/blockingUDPTransport.cpp:260: no more data in UDP packet
```

It seems that there is an older PVA server sending Beacon messages with a different format.  These older beacons are shorter, which trips a check in `ensureData()`.  Previously this exception was being swallowed.

An example of these older beacons

```
0x00 ca000000 21000000 e24e77b2 485c0000
0x10 00002ada a60a0000 00000000 00000000
0x20 ffff0000 0000eaa9 ff
```

And with the current format.

```
0000 ca014000 27000000 5324805c 00000000
0010 7746f033 00000000 00000000 00000000
0020 0000ffff 00000000 d3130374 6370ff
```

**Comment by mdavidsaver** [comment]
The older message uses minor version 0.  I hadn't realized that this was ever a valid version.

**Comment by mdavidsaver** [comment]
Seems it was until 2014 d19cf15a0ec481e52433a9ad7a6980bb952a224c

**Comment by mdavidsaver** [comment]
After looking at 8862d29ea51d149379e0ad8f83b5538ed9b1646d and b101fa1e7a1837194d4b35c176899f9100e2ecd2 there are enough protocol changes that I think it will be better to ignore minor version 0 message.

**Comment by mdavidsaver** [comment]
Included in #144

**Comment by mdavidsaver** [comment]
Released in module version 7.0.0, included in EPICS 7.0.3
